server/cmwell-it/src/it/scala/cmwell/it/APIFunctionalityTests.scala

```diff
@@ -664,6 +664,44 @@ class APIFunctionalityTests extends AsyncFunSpec
         }
       }
 
+      describe("cmwell://meta/sys#markReplace predicate with a wildcard") {
+        it("should ingest multiple fields of two ontologies") {
+          val newData =
+            """
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard/ns#GENDER> "Female" .
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard/ns#FN> "Jane" .
+              |<http://example.org/JaneSmith8> <http://www.w3.org/2006/vcard2/ns#GENDER> "Female" .
+            """.stripMargin
+          Http.post(_in, newData, None, List("format" -> "ntriples"), tokenHeader).map { res =>
+            withClue(res) {
+              jsonSuccessPruner(Json.parse(res.payload)) should be(jsonSuccess)
+            }
+          }
+        }
+        it("should use markReplace with wildcard and verify it removes all relevant fields") {
+          spinCheck(100.millis, true)(Http.get(exampleOrg./("JaneSmith8"), List("format" -> "json"))) { res =>
+            res.status == 200
+          }.flatMap { _ =>
+            val markReplaceData = "<http://example.org/JaneSmith8> <cmwell://meta/sys#markReplace> <http://www.w3.org/2006/vcard/ns#*> ."
```

Comment:
vcard2 still is not in the cache. Even if there is a bug in the new feature, this test won't test it and it might delete all the fields. Am I right?

Reply:
Right. Will fix that one.